### PR TITLE
Display groups im in

### PR DESCRIPTION
**Pull request by WRNoble**
- Related Issue (include '#'): #441

- Description of changes made:

- Is the feature complete/bug resolved/etc..:

- Any known bugs/strange behavior:

- Is there specific feedback you would like on these changes:

- Screenshot(s):

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[[source]]
```
Comment (0xZakk) [inline Pipfile:1]: Can you delete this pipfile?

File: client/src/App.js [hunk 2]
```diff
@@ -37,6 +37,7 @@ import { fas } from "@fortawesome/free-solid-svg-icons";
 import { far } from "@fortawesome/free-regular-svg-icons";
 import { BASE_URL } from "./services/constants";
 import { handleSignup, handleLogin, handleLogout } from "./services/User";
+import GroupPage from "./components/GroupPage/GroupPage"
```
Comment (0xZakk) [inline client/src/App.js:40]: Are you using this page somewhere? Should there be a route for it?

File: client/src/components/GroupPage/GroupPage.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.wrapper {
+    text-align: center;
+}
+
+h1{
+    display: inline-block;
+}
```
Comment (0xZakk) [inline client/src/components/GroupPage/GroupPage.css]: Can you update the CSS here so it's inline with the conventions in the README?

File: client/src/components/GroupPage/GroupPage.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```
Comment (0xZakk) [inline client/src/components/GroupPage/GroupPage.js:1]: Why is this page in the components directory and not the pages directory?

File: client/src/components/GroupPage/GroupPage.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                <h1>Groups You Can Join!</h1>
+            </div>
+            <div className="HomePage__group-cards-container">
+            <span className="HomePage__groupcard-1">
+                <GroupCard
+                name="DIY with your kids"
+                members="98"
+                location="Boston, MA"
+                picture={card1}
+                />
```
Comment (0xZakk) [inline client/src/components/GroupPage/GroupPage.js:32]: Should these be coming from the API?

File: client/src/components/GroupPage/GroupPage.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+            <Navbar 
+            to={"/member-page"} 
+            profileImage={people}
+            />
+            
+            
+            <div className='wrapper'>
+                <h1>Groups You Can Join!</h1>
+            </div>
+            <div className="HomePage__group-cards-container">
```
Comment (0xZakk) [inline client/src/components/GroupPage/GroupPage.js:25]: Where are these classes coming from?

File: client/src/pages/DirectoryPage/DirectoryGroups.js [hunk 7]
```diff
@@ -14,223 +14,196 @@ import picture1 from "../../assets/images/card_small1.png";
...
+    axios.get(`${BASE_URL}/groups`)
+      .then(response => {
+        this.setState({ 
+          groups: response.data
+        })
+      })
+      .catch(error => {
+        console.log('Error while fetching and parsing API request', error);
+      })
+  }
```
Comment (0xZakk) [inline client/src/pages/DirectoryPage/DirectoryGroups.js]: Can you put this in the appropriate service in the `services/` directory? All AJAX requests should be in a service

File: client/src/pages/DirectoryPage/DirectoryGroups.js [hunk 8]
```diff
@@ -14,223 +14,196 @@ import picture1 from "../../assets/images/card_small1.png";
...
+
+  render() {
+    console.log(this.state)
```
Comment (0xZakk) [inline client/src/pages/DirectoryPage/DirectoryGroups.js]: Be sure to delete all commented out code and any `console.log` or `print` statements before making a PR going forward

File: client/src/pages/DirectoryPage/DirectoryGroups.js [hunk 9]
```diff
@@ -14,223 +14,196 @@ import picture1 from "../../assets/images/card_small1.png";
...
+          <div>
+            <header className="CommunityPage__header">Group Location:</header>
           </div>
-          <div className="CommunityPage__groupCard-single">
-            <GroupCard
-              name="Moms traveling to Hawaii"
-              members="5"
-              location="Chicago, IL"
-              picture={picture2}
+          <div className="CommunityPage__group-location-container">
```
Comment (0xZakk) [inline client/src/pages/DirectoryPage/DirectoryGroups.js]: We decided to remove this Pill section right?

File: client/src/pages/DirectoryPage/DirectoryGroups.js [hunk 10]
```diff
@@ -14,223 +14,196 @@ import picture1 from "../../assets/images/card_small1.png";
...
+              onClick={pillClick}
+              selectId={0}
             />
           </div>{" "}
-          <div className="CommunityPage__groupCard-single">
-            {" "}
-            <Card outline size="medium-wide-directory">
-              <a href="/">Discover New Groups</a>
-            </Card>{" "}
+          <a className="seeMore-Button">See More</a>
```
Comment (0xZakk) [inline client/src/pages/DirectoryPage/DirectoryGroups.js]: Can you use names inline with the naming convention described in the README?

File: client/src/pages/DirectoryPage/DirectoryGroups.js [hunk 11]
```diff
@@ -14,223 +14,196 @@ import picture1 from "../../assets/images/card_small1.png";
...
+          <div>
+            <header className="CommunityPage__header">Groups in WASHINGTON, DC:</header>
+          </div>
+          <div className="CommunityPage__groups-in-city-container">
+          {groups.map(group =>{
+            console.log(group)
```
Comment (0xZakk) [inline client/src/pages/DirectoryPage/DirectoryGroups.js]: Delete

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@WRNoble Looks good! See my feedback

**Comment by secondaryfun** [comment]
The updates from this branch have been incorporated into https://github.com/CotripperPlatform/CoTrip/pull/654 .  This branch can be closed without merging.

**Pull request closed without merging**